src/dsp_tools/dsp_tools.py

```diff
@@ -33,149 +32,145 @@ def program(user_args: list[str]) -> None:
         None
     """
     # help texts
-    username_text = 'Username (e-mail) for DSP server'
-    password_text = 'The password for login'
-    url_text = 'URL of the DSP server'
-    verbose_text = 'Verbose feedback'
+    username_text = "username (e-mail) used for authentication with the DSP-API "
+    password_text = "password used for authentication with the DSP-API "
+    url_text = "URL of the DSP server"
+    verbose_text = "print more information about the progress to the console"
 
     # default values
-    default_localhost = 'http://0.0.0.0:3333'
-    default_user = 'root@example.com'
-    default_pw = 'test'
-    dsp_tools_version = version('dsp-tools')
-    now = datetime.datetime.now()
+    default_localhost = "http://0.0.0.0:3333"
+    default_user = "root@example.com"
+    default_pw = "test"
 
     # parse the arguments of the command line
-    parser = argparse.ArgumentParser(description=f'dsp-tools (Version {dsp_tools_version}) DaSCH Service Platform data '
-                                                 f'modelling tools (© {now.year} by DaSCH).')
-    subparsers = parser.add_subparsers(title='Subcommands', description='Valid subcommands are', help='sub-command help')
+    parser = argparse.ArgumentParser(description=f"DSP-TOOLS (version {version('dsp-tools')}, © {datetime.datetime.now().year} by DaSCH)")
+    subparsers = parser.add_subparsers(title="Subcommands", description="Valid subcommands are", help="sub-command help")
 
     # create
```

Comment:
general remark to my modifications:
 - type=str is the default behaviour, so I removed it everywhere
 - some positional arguments (the ones without --) had a default value, which is not possible. I had to remove them